docs/use-mainnet/bridges-of-linea/index.mdx

```diff
@@ -232,3 +232,34 @@ Automatic claiming is only available for ETH L1 to L2 bridging. You will not be
 8. MetaMask will ask you to switch networks and to confirm the transaction from the previous step.
 
 9. You're done! You can check your Ethereum wallet balance on MetaMask or a block explorer such as [etherscan](https://etherscan.io/)!
+
+## How do I bridge ERC20 tokens?
+
+1. Go to the token selection menu.
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/token_selection_menu.png")
+      .default
+  }></img>
+
+
+2. Select one of the available tokens in the list. If the token is not in the list, paste the address of the token you want to add into the search bar.
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/token_list.png")
+      .default    
+  }></img>
+
+
+4. Click on the token to select it.
+
+<img
+  src={
+    require("@site/static/img/docs/use-mainnet/bridges-of-linea/dogecoin.png")
+      .default
+  }></img>
+
```

Comment:
Can you show the before and after states? popup? Is there a video for this too? I find the screenshots unclear